src/components/Sum/Sum.jsx

```diff
@@ -1 +1,6 @@
-// export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
```

Comment:
wrap if you have > 2 params or length > 80
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```